Sources/ParseSwift/Objects/ParseObject.swift

```diff
@@ -16,19 +16,6 @@ import Foundation
  If you are using value types the the compiler will assist you with conforming to `ParseObject` protocol. If you
  are thinking of using reference types, see the warning.
 
- It's recommended the developer adds the emptyObject computed property or similar.
```

Comment:
The documentation replacement should mention that it's recommended developers add the `ParseMutable` protocol to their object and continue with the original reasoning that was already in the documentation.